openwisp_controller/config/static/config/js/widget.js

```diff
@@ -153,6 +153,21 @@
         return error;
     };
 
+    var validateMaxLength = function(){
+        $('#config-group .vTextField').map((index, field) => {
```

Comment:
I think the css selector is too specific, are you sure all the widgets start with `#config-group`? Have you checked devices, templates and vpns?

Reply:
No, It is not working with that. I will fix this.